src/main/java/com/amazon/dlic/auth/http/jwt/keybyoidc/KeySetRetriever.java

```diff
@@ -65,6 +66,16 @@ public class KeySetRetriever implements KeySetProvider {
 		}
 	}
 
+    KeySetRetriever(SSLConfig sslConfig, boolean useCacheForOidConnectEndpoint, String jwksUri) {
```

Comment:
Nit: Can you keep the SSLConfig and cache args. in the same spot of the constructor by placing your `jwksUri` field first?

Reply:
Unfortunately, no. Two constructors/methods with the same name cannot have same parameter types order 😉

Reply:
Doh, haha oops.